source/util/AMErrorMonitor.h

```diff
@@ -161,9 +163,11 @@ class AMErrorMon : public QObject {
 	}
 
 	/// Report an alert level AMErrorReport.  Builds an AMErrorReport using the information provided. This function is thread-safe.
-	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "")
+	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "", bool showWarningMessage=false)
```

Comment:
Let's add it to the other functions. Acquaman is still agile enough that we can plan for the future as we make code changes.